vb6/VisualBasic6Parser.g4

```diff
@@ -0,0 +1,845 @@
+/*
+ * Copyright (C) 2017, Ulrich Wolffgang <ulrich.wolffgang@proleap.io> All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the MIT license. See the LICENSE
+ * file for details.
+ */
+
+/*
+ * Visual Basic 6.0 Grammar for ANTLR4
+ *
+ * This is a Visual Basic 6.0 grammar, which is part of the Visual Basic 6.0 parser at
+ * https://github.com/uwol/vb6parser.
+ *
+ * The grammar is derived from the Visual Basic 6.0 language reference
+ * http://msdn.microsoft.com/en-us/library/aa338033%28v=vs.60%29.aspx and has been tested with MSDN
+ * VB6 statements as well as several Visual Basic 6.0 code repositories.
+ */
+
+parser grammar VisualBasic6Parser;
+options {
+	tokenVocab = VisualBasic6Lexer;
+}
+// module ----------------------------------
+
+startRule: module EOF;
+
+module:
+	WS? NEWLINE* (moduleHeader NEWLINE+)? moduleReferences? NEWLINE* controlProperties? NEWLINE*
+		moduleConfig? NEWLINE* moduleAttributes? NEWLINE* moduleOptions? NEWLINE* moduleBody?
+		NEWLINE* WS?;
```

Comment:
Could you please put an opening colon and closing semi of every parse rule on the next line? It looks more clear.

```suggestion
module
    : WS? NEWLINE* (moduleHeader NEWLINE+)? moduleReferences? NEWLINE* controlProperties? NEWLINE*
		moduleConfig? NEWLINE* moduleAttributes? NEWLINE* moduleOptions? NEWLINE* moduleBody?
		NEWLINE* WS?
    ;
```